### PR TITLE
chore: show error when not enough utxo for checkpoint

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Shows error when you don't have at least 2 utxos for asset creation.

How Has This Been Tested?
---
Manually.

### CODE REVIEW

File: applications/tari_app_grpc/proto/wallet.proto [hunk 1]
```diff
@@ -206,6 +208,10 @@ message GetBalanceResponse {
     uint64 pending_outgoing_balance = 3;
 }
 
+message GetUnspentOutputsResponse {
+    repeated UnblindedOutput outputs = 1;
```
Comment (stringhandler) [inline applications/tari_app_grpc/proto/wallet.proto]: This will expose the blinding factor.... can we provide a slimmer interface

**Review by SWvheerden — COMMENTED** [review]
I am assuming that the whole intention of this PR is to make sure that the wallet has enough funds in enough utxo's to do checkpointing. 

Using the `get_unspent_amounts` call will get you the unspent utxos in the wallet, but those funds may be unspendable as they might be timelocked and or not confirmed.

**Comment by Cifko** [comment]
> I am assuming that the whole intention of this PR is to make sure that the wallet has enough funds in enough utxo's to do checkpointing.
> 
> Using the `get_unspent_amounts` call will get you the unspent utxos in the wallet, but those funds may be unspendable as they might be timelocked and or not confirmed.

The uncofirmed were not there. But the timelocked were. I've added a new method to just get the amounts of utxos that spendable.

**Comment by stringhandler** [comment]
This seems to be implemented in #3802

**Pull request closed without merging**